P5/Source/Guidelines/en/CE-CertaintyResponsibility.xml

```diff
@@ -242,8 +242,8 @@ of the tagging throughout a document, rather than (as discussed so far) in one
 particular instance. The <att>match</att> attribute may be used to
 supply a pattern identifying the portion of a document
 concerning which certainty is being expressed. The value of the
-<att>match</att> attribute is an XPath expression <!--XSLT2 pattern,-->
-using the syntax defined in <ptr target="#XSLT2"/>. In the following
+<att>match</att> attribute is an XPath expression <!--XSLT3 pattern,-->
```

Comment:
I do not think this is correct. I think the value of `@match` is a selection pattern. See [the XSLT spec](https://www.w3.org/TR/xslt-30/#dt-pattern). Thus:
~~~xml
The value of the <att>match</att> attribute is an XSLT selection pattern using the syntax defined in <ptr target="#XSLT3"/>.
~~~